statsd-client/dogstatsd.ml

```diff
@@ -1,217 +1,124 @@
+open Sexplib.Std
+
 let value_or_empty_str f = function
   | None -> ""
   | Some v -> f v
 
-module Tag = struct
-  type t = string * string (** (tag-name, value) *)
-
-  let datagram_fmt l =
-    let format_tag (name, value) = Printf.sprintf "%s:%s" name value in
-    Printf.sprintf "|#%s" (String.concat "," (List.map format_tag l))
+(** https://docs.datadoghq.com/developers/dogstatsd/ *)
+module type T = sig
+  include Dogstatsd_types.Types
 end
```

Comment:
You could just replace this with `module T = Dogstatsd_types.Types`

Reply:
Also (following OCaml conventions), this module should probably be named `S`, not `T`, since interfaces tend to be named `S` and `T` tends to used for concrete types.

Reply:
Ah, interesting. Not having made it through RWO all the way yet, do you happen to know whether it will cover most of these conventsions, or whether there's a good source?

The DRY diff isn't pretty, but it's literally just the body of `T` in both files factored out. So this is the same `T` as before. Combining this with your comment below, would the idea be to do `module T = Dogstatsd_intf.S`, or should it have more appropriately been called `S` prior to this PR?

Reply:
It should probably have been named `S` in the first place.

Unfortunately there's not a great source for the style or conventions. We should probably write one..